src/main/java/uk/gov/hmcts/reform/prl/models/dto/notify/CaseInviteEmail.java

```diff
@@ -4,17 +4,44 @@
 import lombok.Getter;
 import uk.gov.hmcts.reform.prl.models.caseinvite.CaseInvite;
 import uk.gov.hmcts.reform.prl.models.complextypes.PartyDetails;
+import uk.gov.hmcts.reform.prl.models.dto.ccd.CaseData;
+import uk.gov.hmcts.reform.prl.utils.ElementUtils;
+
+import java.util.List;
 
 @Getter
 public class CaseInviteEmail extends EmailTemplateVars {
 
+    private final String caseName;
     private final String accessCode;
     private final String respondentFullName;
+    private final String caseLink;
+    private final String citizenSignUpLink;
+    private final String applicantName;
 
     @Builder
-    public CaseInviteEmail(CaseInvite caseInvite, String caseReference, PartyDetails party) {
+    public CaseInviteEmail(CaseInvite caseInvite, String caseReference, PartyDetails party,
+                           String caseLink, String citizenSignUpLink, CaseData caseData) {
         super(caseReference);
         this.accessCode = caseInvite.getAccessCode();
         this.respondentFullName = String.format("%s %s", party.getFirstName(), party.getLastName());
+        this.caseLink = caseLink;
+        this.citizenSignUpLink = citizenSignUpLink;
+        this.caseName = caseData.getApplicantCaseName();
+        this.applicantName = getApplicantNames(caseData);
+    }
+
+    private static String getApplicantNames(CaseData caseData) {
+        List<PartyDetails> applicants = ElementUtils.unwrapElements(caseData.getApplicants());
+        String applicantNames = "";
```

Comment:
Can we move this to stringBuffer please?

Reply:
Updated to use StringBuilder - hope thats ok?